vllm/v1/executor/abstract.py

```diff
@@ -62,14 +62,11 @@ def initialize_from_config(self,
                             args=(kv_cache_configs, ))
         self.collective_rpc("compile_or_warm_up_model")
 
-    def determine_available_memory(self) -> int:  # in bytes
+    def determine_available_memory(self) -> List[int]:  # in bytes
```

Comment:
+1 why do we need to change the return type of this function?

Reply:
https://github.com/vllm-project/vllm/pull/14079/files#r1986211589